yb-voyager/cmd/export.go

```diff
@@ -61,7 +62,7 @@ func registerSourceDBConnFlags(cmd *cobra.Command, includeOracleCDBFlags bool) {
 	cmd.Flags().StringVar(&source.Host, "source-db-host", "localhost",
 		"source database server host")
 
-	cmd.Flags().IntVar(&source.Port, "source-db-port", -1,
+	cmd.Flags().IntVar(&source.Port, "source-db-port", 0,
```

Comment:
Curious, why change from -1 -> 0? -1 seems more indicative that it is a placeholder, not an actual default, right?

Reply:
Default as -1 was displayed on the CLI help as `default(-1)` but if we use 0, it will not be displayed and we use only our msg to display the default.

Reply:
Ah, perfect!